src/panel.css

```diff
@@ -1,51 +1,248 @@
+  padding-bottom: 1rem;
+}
+
+.main-content-wrapper a:last-child {
+  padding-bottom: 0;
 }
 
-.userContext-icon {
-  padding-left: 25px;
-  padding-right: 45px;
+h1 { /* "Facebook Container" headline for all panels */
+  font-weight: 600;
+  border-bottom: var(--borders);
+  position: relative;
   display: flex;
   align-items: center;
+  justify-content: center;
+  min-height: 46px;
 }
 
-.userContext-label {
-  margin-inline-start: 0;
-  margin-inline-end: 3px;
-  color: #37adff;
+.btn-return.arrow-left {
+  position: absolute;
+  border: 0;
+  left: 15px;
+  top: 15px;
+  width: 1.2rem;
+  height: 1.2rem;
+  background-color: rgba(255, 255, 255, 1);
+  background-image: url("/img/arrow-icon-left.svg");
+  cursor: pointer;
 }
 
-.userContext-indicator {
-  width: 16px;
-  height: 16px;
-  background-image: url("briefcase.svg");
-  background-size: contain;
-  background-repeat: no-repeat;
+h2 { /* panel sub-head for all panels */
+  display: flex;
+  align-items: center;
+}
+
+h2,
+.bold {
+  font-weight: 500;
+}
+
+.Facebook-text {
+  color: var(--secondaryText);
+  font-weight: 500;
+  font-size: 14px;
+}
+
+.Facebook-text::after { /* grey Facebook with fence */
+  background-image: url("/img/fbc-icon2.svg");
+  display: inline-block;
+  width: 14px;
+  height: 12px;
+  margin-left: 4px;
+  margin-bottom: -1px;
+}
+
+p,
+a {
+  font-weight: 400;
+  line-height: 140%;
+}
+
+a {
+  color: var(--blue60);
+  text-decoration: none;
+  cursor: pointer;
+}
+
+a:hover {
+  color: var(--blue70);
+  border-bottom: 1.5px solid var(--blue70);
+}
+
+.fw-bottom-btn { /* Full-width bottom bar "How Facebook Container Works" button. Only on non-onboarding panels. */
+  padding: .7rem 0;
+  background-color: transparent;
+  border-top: var(--borders);
+}
+
+.highlight-on-hover { /* Full-width element that wraps around the "How Faceook Container Works" span and arrow icon */
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0);
+  border: 0;
+  cursor: pointer;
+  padding: .45rem 1.65rem;
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  transition: var(--transition);
+}
+
+.highlight-on-hover::after {
+  position: absolute;
+  display: inline-block;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+  right: 15px;
+  height: 100%;
+  width: .5rem;
+  background-image: url("/img/arrow-icon-right.svg");
   background-position: center center;
+  transform: scale(1.3);
 }
 
-.img {
-  padding-right: 10px;
-  align-items: baseline;
+.highlight-on-hover span {
+  pointer-events: none;
 }
-  
-.heading {
+
+.highlight-on-hover:hover {
+  background-color: var(--grey20);
+  transition: var(--transition);
+}
+
+.fw-bottom-btn,
+.bottom-btns {
+  display: flex;
+  justify-content: space-between;
+  transition: var(--transition);
+  cursor: pointer;
+}
+
+.bottom-btns { /* Bottom "Cancel, Back, Next, Done" buttons - onboarding panels only */
+  width: 100%;
+}
+
+.bottom-btn {
+  border-bottom: 0;
+  line-height: 103%;
+  height: 3.37rem;
```

Comment:
Recommend changing this a static pixel height of `45px`. It should resolve the additional class of `.onboarding3 .bottom-btn`.